examples/jira-scan-workflow/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,562 @@
+                  value: '{{ inputs.parameters.JIRA_API_KEY_SECRET_KEY }}'
+                - name: JIRA_ISSUE_SOURCE_FIELD
+                  value: "{{tasks.create-jira.outputs.parameters.JIRA_ISSUE_SOURCE_FIELD}}"
+                - name: DESIRED_ISSUE_STATUS
+                  value: 'Completed'
+                - name: VERBOSE
+                  value: '{{ inputs.parameters.VERBOSE }}'
+
+    # test template
+    - name: test
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Test image pulling correctly and printing it content files'
+          argo-hub-template/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/jira-scan-workflow/assets/icon.svg"
+          argo-hub-template/icon_background: "#f4f4f4"
+      inputs:
+        parameters:
+          - name: IMAGE
+      container:
+        image: '{{ inputs.parameters.IMAGE }}'
+        command: [sh, -c]
+        args: ["ls"]
+
+    # canary rollout template 10%
+    - name: rollout-10
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Applies a 10% canary rollout'
+          argo-hub-template/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/jira-scan-workflow/assets/icon.svg"
+          argo-hub-template/icon_background: "#f4f4f4"
+      inputs:
+        artifacts:
+          - name: REPO
+            path: /tmp/repo
+        parameters:
+          - name: IMAGE
+      container:
+        image: codefresh/cf-deploy-kubernetes:master
+        workingDir: /tmp/repo
+        commands:
+          - /cf-deploy-kubernetes ./canary-manual-approval/service.yaml 
+          - /cf-deploy-kubernetes ./canary-manual-approval/service-preview.yaml
+          - /cf-deploy-kubernetes ./canary-manual-approval/service-all.yaml  
+          - /cf-deploy-kubernetes ./canary-manual-approval/rollout.yaml   
+
+    # canary rollout template 33%
+    - name: rollout-33
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Applies a 33% canary rollout'
+          argo-hub-template/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/jira-scan-workflow/assets/icon.svg"
+          argo-hub-template/icon_background: "#f4f4f4"
+      inputs:
+        artifacts:
+          - name: REPO
+            path: /tmp/repo
+        parameters:
+          - name: IMAGE
+      container:
+        image: kostiscodefresh/kubectl-argo-rollouts:latest
+        workingDir: /tmp/repo
+        commands:
+          - /app/kubectl-argo-rollouts-linux-amd64 promote golang-sample-app-deployment -n canary --context "mydemoAkscluster@BizSpark Plus"
+    
+    # canary rollout template 100%
+    - name: rollout-100
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Applies a 100% canary rollout'
+          argo-hub-template/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/jira-scan-workflow/assets/icon.svg"
```

Comment:
need to be change to this "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/jira-scan-workflow/assets/icon.svg"